solidity/contracts/bridge/BridgeState.sol

```diff
@@ -375,16 +383,22 @@ library BridgeState {
     ///        be incurred by each swept deposit being part of the given sweep
     ///        transaction. If the maximum BTC transaction fee is exceeded,
     ///        such transaction is considered a fraud.
+    /// @param _depositRevealAheadPeriod New value of the deposit reveal ahead
+    ///        period parameter in seconds. It defines the length of the period
+    ///        that must be preserved between the deposit reveal time and the
+    ///        deposit refund locktime.
     /// @dev Requirements:
     ///      - Deposit dust threshold must be greater than zero,
     ///      - Deposit dust threshold must be greater than deposit TX max fee,
     ///      - Deposit treasury fee divisor must be greater than zero,
     ///      - Deposit transaction max fee must be greater than zero.
+    ///      - Deposit reveal ahead period must be grater than zero.
```

Comment:
The description of `depositRevealAheadPeriod` field says that:

> The value equal to zero disables the validation of this parameter.

Also, it does not look like we have a non-zero validation rule for this field.

Reply:
See 95a1f0f9